ocs_ci/deployment/metallb.py

```diff
@@ -698,3 +705,85 @@ def apply_icsp(self):
         wait_for_machineconfigpool_status(node_type="all")
         logger.info("ICSP applied successfully")
         return self.icsp_brew_registry_exists()
+
+    def metallb_patch_subscription(self, patch):
+        patch_cmd = (
+            f"oc -n {self.namespace_lb} patch sub {constants.METALLB} "
+            f"-p {patch} --type merge"
+        )
+        run_cmd(patch_cmd)
+
+    def get_running_metallb_version(self):
+        """
+        Get the currently deployed cnv version
+
+        Returns:
+            string: metalLB version
+
+        """
+        occmd = f"oc get sub {constants.METALLB} -n {self.namespace_lb} -o json"
+        jq_cmd = "jq -r .status.currentCSV"
+        json_out = subprocess.Popen(shlex.split(occmd), stdout=subprocess.PIPE)
```

Comment:
this can be replaced with `exec_cmd(cmd, shell=True)`

Reply:
Addressed.